### PR TITLE
build(*): update ng to @9.0.0

### LINKED ISSUE #6409: Update peer dependencies in library package.json to Angular 9
## Description  
The versions of the angular peer dependencies of the 9 beta packages are still using `"^8.0.0"` https://cdn.jsdelivr.net/npm/igniteui-angular@9.0.0-beta.6/package.json

 * igniteui-angular version:  9.0.0-beta.6
 * browser: N/A

### PR DESCRIPTION
**Pull request by Lipata**
Closes #6409 

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by damyanpetev** [comment]
And close #6409 in that case

**Comment by Lipata** [comment]
Closing https://github.com/IgniteUI/igniteui-angular/pull/6600 because, I've added change ```@igniteui/angular-schematics": "^9.0.500``` in that pull request

File: projects/igniteui-angular/package.json [hunk 1]
```diff
@@ -72,14 +72,14 @@
...
-        "@angular/forms": "^8.0.0",
+        "@angular/common": "^9.0.0",
+        "@angular/core": "^9.0.0",
+        "@angular/animations": "^9.0.0",
+        "@angular/forms": "^9.0.0",
         "web-animations-js": "^2.3.2"
     },
     "devDependencies": {
-        "@igniteui/angular-schematics": "^9.0.500-rc.1"
+        "@igniteui/angular-schematics": "^9.0.500"
```
Comment (damyanpetev) [inline projects/igniteui-angular/package.json]:
```suggestion
        "@igniteui/angular-schematics": "~9.0.500"
```

File: projects/igniteui-angular/package.json [hunk 2]
```diff
@@ -72,14 +72,14 @@
...
-        "@angular/forms": "^8.0.0",
+        "@angular/common": "^9.0.0",
+        "@angular/core": "^9.0.0",
+        "@angular/animations": "^9.0.0",
+        "@angular/forms": "^9.0.0",
         "web-animations-js": "^2.3.2"
     },
     "devDependencies": {
-        "@igniteui/angular-schematics": "^9.0.500-rc.1"
+        "@igniteui/angular-schematics": "~9.0.500"
```
Comment (damyanpetev) [inline projects/igniteui-angular/package.json]: Actually if this is to be included in the final RC, revert back to a matching RC of the CLI

**Pull request merged**